etc/DependencyInstaller.sh

```diff
@@ -86,20 +86,33 @@ _installUbuntuPackages() {
     if _versionCompare $1 -ge 23.04; then
         apt-get install klayout python3-pandas
     else
-        if [[ $1 == 20.04 ]]; then
-            klayoutChecksum=15a26f74cf396d8a10b7985ed70ab135
-        else
-            klayoutChecksum=db751264399706a23d20455bb7624264
-        fi
+        arch=$(uname -m)
         lastDir="$(pwd)"
         # temp dir to download and compile
         baseDir=/tmp/installers
         mkdir -p "${baseDir}"
-        cd ${baseDir}
-        wget https://www.klayout.org/downloads/Ubuntu-${1%.*}/klayout_${klayoutVersion}-1_amd64.deb
-        md5sum -c <(echo "${klayoutChecksum} klayout_${klayoutVersion}-1_amd64.deb") || exit 1
-        dpkg -i klayout_${klayoutVersion}-1_amd64.deb
-        cd ${lastDir}
+        cd "${baseDir}"
+        if [[ $arch == "aarch64" ]]; then
+        if [ ! -f /usr/local/bin/klayout ]; then
+            echo "Installing KLayout for aarch64 architecture"
+            installDir="/usr/local/bin"
```

Comment:
I don't think this should default to `/usr/local/bin`, it should default to where the other packages from source are built.